crates/libcontainer/src/config.rs

```diff
@@ -0,0 +1,44 @@
+use std::{
+    fs,
+    path::{Path, PathBuf},
+};
+
+use anyhow::{Context, Result};
+use serde::{Deserialize, Serialize};
+
+use oci_spec::runtime::{Hooks, Spec};
+
+use crate::utils;
+
+// TODO: comments and examples
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct YoukiConfig {
+    pub hooks: Option<Hooks>,
+    pub cgroup_path: PathBuf,
+}
```

Comment:
Do you think we should add `non_exhaustive` attribute here? That way if any external crate depends on this, we can extend this without breaking it. 
Suggesting, as this is quite likely to extend with other required fields.

Reply:
I didn't know it. Nice idea!
I added it.